.jenkinsci/artifacts.groovy

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env groovy
+
+def uploadArtifacts(filePaths, uploadPath, artifactServers=['artifact.soramitsu.co.jp']) {
+  def baseUploadPath = 'files'
+  def filePathsConverted = []
+  agentType = sh(script: 'uname', returnStdout: true).trim()
+  uploadPath = baseUploadPath + uploadPath
+  filePaths.each {
+    fp = sh(script: "for i in \$(ls -d ${it}); do FILES=\${i},\${FILES};done;echo \$FILES", returnStdout: true).trim()
```

Comment:
Is there any problem with `tr`?

Reply:
What do you mean? Replace `trim()` with `tr -d`?

Reply:
no, I mean shell script replacement (for + concatenation)